pubspec.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
   clock: ^1.1.1
   collection: ^1.17.0
   connectivity_plus: ^7.0.0
-  cronet_http: ^1.3.1
+  cronet_http: 1.6.0
```

Comment:
Why this change? It has nothing to do with the PR. Please remove it (also it fixes the version which we don't want).